servers/src/main/java/tachyon/master/InodeFolder.java

```diff
@@ -103,6 +110,20 @@ public InodeFolder(String name, int id, int parentId, long creationTimeMs) {
   }
 
   /**
+   * Create a new InodeFolder.
+   *
+   * @param name The name of the folder
+   * @param id The inode id of the folder
+   * @param parentId The inode id of the parent of the folder
+   * @param creationTimeMs The creation time of the folder, in milliseconds
+   * @param permissionStatus the directory permission information
+   */
+  public InodeFolder(String name, int id, int parentId, long creationTimeMs,
```

Comment:
I think using a new construct method is a better idea. Because it doesn't need to change a lot of original code.